docs/src/map_with_inverse.md

```diff
@@ -106,8 +106,11 @@ Map with section
 
 julia> h = f*g
 Composite map
-  first map: integers -> integers
-  next map: integers -> integers
+  from integers
+  to integers
+which is the composite of
+  Map: integers -> integers
+  Map: integers -> integers
```

Comment:
This change is admittedly a bit gratuitous, I just made it so that the printing better fits in... If you dislike it, @simonbrandhorst I can also revert this bit or adjust it further.